### PR TITLE
Fix relative paths for kubeflow instructions

### PR DESCRIPTION
**Pull request by yanrez**
Issues fixed:
- KUBEFLOW_SOURCE was used instead of KUBEFLOW_SRC
- As user moves into subfolders, ${KUBEFLOW_SRC} wouldn't work either unless KUBEFLOW_SRC has absolute path

### CODE REVIEW

File: 4-kubeflow/README.md [hunk 1]
```diff
@@ -50,15 +50,15 @@ curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts
 
 ```bash
 # Initialize a kubeflow app
-KFAPP=mykubeflowapp
-${KUBEFLOW_SOURCE}/scripts/kfctl.sh init ${KFAPP} --platform none
+KFAPP=$(realpath mykubeflowapp)
```
Comment (ritazh) [inline 4-kubeflow/README.md]: This should be a name not a path. 
https://github.com/kubeflow/kubeflow/blob/master/scripts/kfctl.sh#L342

**Review by ritazh — CHANGES_REQUESTED** [review]
Thanks @yanrez! Just one minor comment.

File: 4-kubeflow/README.md (hunk 1 above)
Comment (yanrez) [reply 4-kubeflow/README.md]: fixed

**Review by ritazh — APPROVED** [review]

**Comment by sozercan** [comment]
thanks @yanrez!

**Pull request merged**